fossunited/foss_hackathon/doctype/foss_hackathon_participant/foss_hackathon_participant.json

```diff
@@ -108,6 +108,8 @@
    "fieldname": "localhost_request_status",
    "fieldtype": "Select",
    "label": "LocalHost Request Status",
+   "options": "Pending\nAccepted\nRejected",
+   "permlevel": 1
```

Comment:
We need to revert changes to this file. The changes are unrelated and don't need to be included in this PR.

```suggestion
```